### ISSUE TITLE
Remove Rotating Cities. Go back to Standard List.

### ISSUE THREAD

**Issue by Deceth** [Deceth/Battle-City#7]
**v4.0.0**
- Remove rotating cities. Leave functionality in-tact, but disable.
- Have 9 starting cities. Open more as more players log-on.
- Allow players to gain permanent access keys for any city via donation. Database will need to be updated to support this. Currently player can only have access to one city at a time. Must be made unlimited.

> Rushing said: 
> 
> "I think this one is an important one. Twice I heard this, or something akin to this being said;
> 
> 'Durr, I have no cities saved for any of these' 
> 
> This I have now realized, is a rather large problem. New players, generally join more than once at least. When they rejoin, they see new cities and thus need to start over again. I hope you put it back to the Normal starting cities, and have them increase as more players get on."

**Comment by Deceth** [comment]
I agree. In order to finance my evil operations it would be in my best interest to turn off rotating cities and allow my minions to donate in order to gain access to some of the more extravagant cities out there.

The benefit of rotating cities was to increase players because people were bored with the same old cities. Obviously that failed or I would not be instructing my engineers to point a giant laser at the Battle City Server as we speak...

**Comment by Rushing** [comment]
I think it was a success, but I think it's only a success if there is a large populous on the game already.

**Comment by KiL3R** [comment]
Yeah. Stop the rotating cities. Deceth, glad to have you still. working on BC :)

**Comment by Alton** [comment]
Hm. someone make a time machine to go back to 2006! :D! ...i wish i could...i miss how bc had tons of players on it..that was the best time ever.....

**Comment by Deceth** [comment]
Just to update this:

**v4.0.0**
- Remove rotating cities. 
- Have 9 starting cities. Open more as more players log-on.
- Allow players to gain access for any city via donation.

**Remove rotating cites**
There is a function set in the admin panel that allows an administrator to set the starting city. Once the starting city is set, this determines which 9 cities are open for players to join. When the server population reaches 0, this variable changes automatically, so the set of available cities is always changing.

This was tried for a while, but the preference was for a stable set of 9 starting cities as discussed next. Therefore, remove the rotating cities functionality.

**Have 9 starting cities. Open more as more players log-on.**

The following 9 cities should be open by default
- Andorra-la-Vella
- Benguela
- Bahia Blanca
- Santa Fe 
- Buenos Aires
- Cordoba
- Posadas
- Resistencia
- Rio Cuarto

As more players logon, additional cities will become available. The above creates a 3x3 square, so the next iteration should expand such that the total availability now forms a 4x4 square, then 5x5 square, etc...

**Allow players to gain access for any city via donation.**
Database will need to be updated to support this by creating a new table that stores a player ID and city ID. Functionality already exists to check if a player has a "rental city", but we want to allow players to have an unlimited amount. If a player has a rental city, it is available at all times on the list of cities they can join (unless someone else is using it) 

The purpose here is to give something extra to those who help pay for a dedicated server, without giving them an unfair advantage in battle.
